event/PageSaveEvent.php

```diff
@@ -31,9 +31,16 @@ public function getEventType(): ?string
 
     /**
      * Root namespace of the page.
+     * Handle case of page being in root namespace
      */
     public function getNamespace(): string
     {
+        $pos = strpos($this->id, ':');
+        if($pos === false)
+        {
+            return ('');
```

Comment:
```suggestion
            return '';
```

you mean empty string here? the no need for parens, you want array, then it should be `['']`

Reply:
Fixed I think

Reply:
missing link to commit:
- 39c3047e4ea2779c6a6826014e3c50f5c7c705ed